app/models/role.rb

```diff
@@ -5,12 +5,12 @@ class Role < ApplicationRecord
   belongs_to :parent, class_name: 'Role', optional: true
   has_many :users, dependent: :nullify
 
-  # Role IDs
-  STUDENT_ID = 1
-  TEACHING_ASSISTANT_ID = 2
+  # Role IDs (must match db/seeds.rb)
+  SUPER_ADMINISTRATOR_ID = 1
```

Comment:
Could this just be SUPER_ADMINISTRATOR, etc.?  All files would need to be refactored to effect this change, but it would enhance readability.  But, would it lead to namespace conflicts?  If it doesn't cause name conflicts, it would be worth doing.